Fix CI build

Last success build: https://github.com/fcitx/fcitx5-cskk/actions/runs/11668965674

Failure since: https://github.com/fcitx/fcitx5-cskk/actions/runs/13194988545

Investigate and fix the build of github actions.

The main issue is that github image couldn't prepare the required library dev packages. But we have updated the least required version and I'm not sure if any of github images can prepare the fcitx5 dev packages. At worst, I need to stop this build-check on the CI pipeline and wait for the ubuntu  to adopt the latest fcitx5 libraries.


```
-- Found PkgConfig: /usr/bin/pkg-config (found version "1.8.0")
  CMake Error at CMakeLists.txt:24 (find_package):
    Could not find a configuration file for package "Fcitx5Core" that is
    compatible with requested version "5.1.13".
  
    The following configuration files were considered but not accepted:
  
      /usr/lib/x86_64-linux-gnu/cmake/Fcitx5Core/Fcitx5CoreConfig.cmake, version: 5.0.14
      /lib/x86_64-linux-gnu/cmake/Fcitx5Core/Fcitx5CoreConfig.cmake, version: 5.0.14
  ```


According to https://github.com/actions/runner-images, the latest image is ubuntu-24.04 
but even on ubuntu25, ubuntu's latest fcitx package is 5.1.9. And also I don't see Fcitx5Qt6Widgets (fcitx5-frontend-qt6-dev or alike if package exists?)

I'd give up this build check on the github actions and see if we can enable back when github actions update the image.



Or, when I or someone have time, making a reusable action to setup the build environment like the one in check.yml and use that for the disabled action might be also an solution.

--- a/.github/workflows/build_pass.yml
+++ b/.github/workflows/build_pass.yml
@@ -1,6 +1,8 @@
 name: Build passes check
-on: [ push ]
-
+# Disabled for push for now.
+# Re-enable when github actions has good image for build environment. (Issue#83)
+#on: [ push ]
+on: workflow_dispatch
 concurrency:
   group: "continuous_test"
   cancel-in-progress: true
@@ -8,7 +10,7 @@ concurrency:
 jobs:
   build-check:
     name: Build check
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     steps:
       - name: checkout
         uses: actions/checkout@v3

--- a/.github/workflows/check.yml
+++ b/.github/workflows/check.yml
@@ -100,4 +100,4 @@ jobs:
         run: |
           ctest --test-dir fcitx5-cskk/build
       - name: CodeQL Analysis
-        uses: github/codeql-action/analyze@v2
+        uses: github/codeql-action/analyze@v3

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -1,11 +1,15 @@
 name: Release x86_64 deb artifact on release published
-on:
-  release:
-    types:
-      - published
+# Disabled for release for now.
+# Re-enable when github actions has good image for build. (Issue#83)
+#on:
+#  release:
+#    types:
+#      - published
+on: workflow_dispatch
+
 jobs:
   Build:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
 #    strategy:
 #      matrix:
 #        cskk_ver: [0.8.1]

